src/main/java/io/github/perrymant/moneymaker/ReportRepository.java

```diff
@@ -5,14 +5,16 @@
 import java.io.File;
 import java.io.IOException;
 
-class AccountRepository {
+class ReportRepository {
```

Comment:
We should probably add a `@Component` or similar on this. Make sure the spring app still starts up after doing that.